packages/sdk/src/types/relationships/ConnectorRelationship.ts

```diff
@@ -12,6 +12,6 @@ export interface ConnectorRelationship {
     status: ConnectorRelationshipStatus;
     peer: string;
     peerIdentity: ConnectorIdentity;
-    creationContent: ConnectorRelationshipCreationContentDerivation;
+    creationContent: RelationshipCreationContentJSON | ArbitraryRelationshipCreationContentJSON;
```

Comment:
We could still define RelationshipCreationContentDerivation = RelationshipCreationContentJSON | ArbitraryRelationshipCreationContentJSON, since it is used in multiple places.

Reply:
see above